WikiPreview.Fluent.Plugin/WikiPreviewSearchOperation.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Blast.Core.Results;
+
+namespace WikiPreview.Fluent.Plugin
+{
+    class WikiPreviewSearchOperation
+    {
+
+        public class WikiSearchOperation : SearchOperationBase
```

Comment:
No reason to have multiple types of operations if they don't have special properties.
Add an `enum` that describes your operation and put it as a property of `WikiSearchOperation`, that way you can do a `switch` on the enum in `HandleSearchResult`